crypto/merkle/merkle.proto

```diff
@@ -4,7 +4,7 @@ option go_package = "github.com/tendermint/tendermint/crypto/merkle";
 
 // For more information on gogo.proto, see:
 // https://github.com/gogo/protobuf/blob/master/extensions.md
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "third_party/proto/gogoproto/gogo.proto";
```

Comment:
I would include `third_party/proto` as a root (and also include for `protoc` with `-I third_party/proto`), and remove that part from these imports @marbar3778

Reply:
added `-I third_party/proto` but not sure what you mean by add it to root. If in the buf.yml file then it overlaps with `.`